shared/constants/wallets.tsx

```diff
@@ -171,7 +216,9 @@ export const makeState = I.Record<Types._State>({
   assetsMap: I.Map(),
   buildCounter: 0,
   building: makeBuilding(),
+  buildingAdvanced: emptyBuildingAdvanced,
```

Comment:
Not sure why these exist.. using makeBuildingAdvanced() directly seems like it's more consistent and doesn't take up more space.

Reply:
We use this in some of the `get`s too, so having a constant means getting same result from state selectors in those cases, if they remain empty.